e2e-tests/cypress/support/appeal-navigation-confirmation/appellant-submission/confirmNavigationApplicantNamePage.js

```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('include', '/appellant-submission/applicant-name');
+  cy.url().should('include', '/appeal-householder-decision/applicant-name');
```

Comment:
please can this dir be renamed from `appellant-submission` to `appeal-householder-decision` ? thanks